docs/css/tc-header.css

```
@@ -0,0 +1,375 @@
+    --color-topaz: #00BAC2;
+    --color-pacific: #17A6B2;
+    --color-teal: #027F9E;
+    --color-eggplant: #291A3F;
+    --color-plum: #361E5B;
+
+}
+
+#site-header {
+    color: var(--color-storm);
+    background: #fff;
+    font-family: 'Rubik', Arial, Helvetica, sans-serif;
```

Stylelint (17.3.0)

[error] Unexpected quotes around "Rubik" (font-family-name-quotes)

(font-family-name-quotes)